.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+python:
+  - "3.6"
+install:
+  - wget https://github.com/ethereum/solidity/releases/download/v0.4.18/solc-static-linux
+  - chmod +x ./solc-static-linux
+  - sudo mv solc-static-linux /usr/bin/solc
+  - pip install -r requirements.txt
+  - npm install -g ganache-cli
+  - ganache-cli -m=plasma_cash > /dev/null &
```

Comment:
integration test would spin up ganache + deploy + child chain server automatically, probably don't need this & the following lines.